Islandora/islandora
# Include islandora_fits into islandora/modules

**GitHub Issue**: https://github.com/Islandora/islandora/issues/906

# What does this Pull Request do?

Includes Islandora Fits as a submodule of Islandora.

# What's new?

* Islandora Fits submodule, including its git history.
* Added a dependency to `composer.json` for ext-simpleXML, required by Islandora FITS so it can parse XML.

* Does this change add any new dependencies? - yes, simpleXML. will be installed automatically when using Composer to update islandora.
* Does this change require any other modifications to be made to the repository
 (i.e. Regeneration activity, etc.)? 
* Could this change impact execution of existing code? no, shouldn't.

# How should this be tested?

To upgrade to this version:
* add this branch as a repository in your composer.json, and change the spec for "islandora/islandora" to "dev-include-fits" (won't be required after merging).
* `composer remove "islandora-rdm/islandora_fits"`
* `composer update -W "islandora/islandora"` (make sure you get the new code)
* cache refresh and restart your webserver so PHP finds the files in their new locations.
* FITS derivatives should work exactly as before.
* Note: I did NOT merge the `islandora_fits_tags` migration with the islandora tags migration. I'm not sure we should, since this is an optional module. But i'm open to suggestions.

# Documentation Status

* Does this change existing behaviour that's currently documented? - no, same behaviour
* Does this change require new pages or sections of documentation? - no
* Who does this need to be documented for? - n/a
* Associated documentation pull request(s): ___  or documentation issue ___

# Deployment Status (post-commit)

* Starter Site - will need an update (removing islandora_fits from composer when locking to this version of islandora)
* Install Profile - will need an update (removing islandora_fits)

# Additional Notes:
Any additional information that you think would be helpful when reviewing this
 PR.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora/committers
